Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:2.3.3
```

Comment:
the `.ruby-version` file is using 2.2.5.  idk what this does but it seems like it should match.  we were going to use 2.4.0, but there's some bug with 2.4.0 on macs.  you could probably just update the ruby-version